src/main/kotlin/uk/co/craigbass/pratura/usecase/ViewBasket.kt

```diff
@@ -18,7 +18,15 @@ class ViewBasket(private val basketItemsGateway: BasketItemsRetriever,
     return PresentableBasket(lineItems(), "£${getBasketValue()}")
   }
 
-  private fun lineItems() = items!!.map { item -> PresentableLineItem(item.quantity, item.sku) }
+  private fun lineItems(): List<PresentableLineItem> {
+    return items!!.map { item ->
```

Comment:
yeah this will throw an exception if `items` is null. It's fine, I have a test suite :smile: